docs/storefront/graphql/examples/index.mdx

```diff
@@ -223,3 +223,27 @@ query Favicon {
 ```
 
 <a href="https://developer.bigcommerce.com/graphql-storefront/playground" target="_blank">**Try it in GraphQL Playground**</a>
+
+## Get analytics ID
+
```

Comment:
I would add some note that when some of those analytics are configured for store their ids can be retrieved using this query. 
Maybe linking to some analytics configuration docs would also be helpful.